src/elements/ngm-side-bar.ts

```diff
@@ -320,6 +330,11 @@ export class SideBar extends LitElementI18n {
       }
     }
 
+    if (this.querySelector('#langs') && !this.langInited) {
+      setupI18n();
+      this.langInited = true;
+    }
+
```

Comment:
I don't think you need this. The component should re-render automatically once i18n is initialized (initialization is already triggered in ngm-app).

Reply:
ok, reverted so previous initialization, seems to work fine.